README.md

```diff
@@ -56,7 +56,7 @@ The above figure demonstrates the core OpenYurt architecture. The major componen
   `YurtAppDaemon` enables to deploy workload for every matching NodePool.
   `YurtIngress` orchestrates the deployment of multiple ingress controllers to the respective NodePools.
 - **[Raven/Raven Controller Manager](https://openyurt.io/docs/next/core-concepts/raven)**: It is focused on edge-edge and edge-cloud communication in OpenYurt. and provide layer 3 network connectivity among pods in different physical regions, as there are in one vanilla Kubernetes cluster.
-- **Pool-Coordinator**: On instance of Pool-Coordinator is deployed in every edge NodePool, and in conjunction with YurtHub to provide heartbeat proxy, cloud edge traffic reuse abilities, etc.
+- **Pool-Coordinator**: One instance of Pool-Coordinator is deployed in every edge NodePool, and in conjunction with YurtHub to provide heartbeat delegation, cloud edge traffic reuse abilities, etc.
```

Comment:
We originally call `cloud edge traffic reuse` as `cloud-edge traffic multiplexing` in proposal.

Reply:
It seems to use `cloud-edge traffic multiplex` will be better, since multiplex can be noun.

Reply:
fixed